src/TwigComponent/src/Twig/ComponentNode.php

```diff
@@ -154,16 +154,29 @@ public function compile(Compiler $compiler): void
         if ($useYield) {
             $compiler->write('yield from ');
         }
-        $compiler
-            ->write('$this->loadTemplate(')
-            ->string($this->getAttribute('embedded_template'))
-            ->raw(', ')
-            ->repr($this->getTemplateName())
-            ->raw(', ')
-            ->repr($this->getTemplateLine())
-            ->raw(', ')
-            ->string($this->getAttribute('embedded_index'))
-            ->raw(')');
+
+        if (Environment::VERSION_ID >= 32100) {
+            $compiler
+               ->write('$this->load(')
+               ->string($this->getAttribute('embedded_template'))
+               ->raw(', ')
+               ->repr($this->getTemplateLine())
+               ->raw(', ')
+               ->string($this->getAttribute('embedded_index'))
+               ->raw(')');
+        } else {
+            // @deprecated since Twig 3.20
```

Comment:
Either the comment should say `deprecated since Twig 3.21` (or `compat for Twig <= 3.20 instead), or the version_id check is using the wrong version.